content/en/community/contribution.md

```diff
@@ -0,0 +1,25 @@
+--- 
+title: "CHT Squads" 
+linkTitle: "Squads"
+weight: 2 
+description: > 
+  What is a contribution?
+---
+
+**What is a contribution?**
```

Comment:
<img width="297" height="113" alt="Screenshot 2025-09-02 at 09 50 30" src="https://github.com/user-attachments/assets/f6da2a2c-425c-4ebb-a248-05c32e0ea24d" />

There is a duplicate question that feels unnecessary. Please review the wording to remove duplication.